src/market/types.py

```diff
@@ -80,7 +81,47 @@ class Order(NamedTuple):
     """"""
     fee_tier: int
     """"""
-    order_info: OrderInfo
+    info: OrderInfo
     """"""
     side: Side
     """"""
+
+
+class Request(NamedTuple):
+    request_flags: Container  # TODO: Remove container type
+    """"""
+    open_order_slot: int
+    """"""
+    fee_tier: int
+    """"""
+    max_base_size_or_cancel_id: int
+    """"""
+    native_quote_quantity_locked: int
+    """"""
+    order_id: int
+    """"""
+    open_orders: PublicKey
+    """"""
+    client_order_id: int
+    """"""
+
+
+class Event(NamedTuple):
+    event_flags: Container  # TODO: Remove container type
+    """"""
+    open_order_slot: int
+    """"""
+    fee_tier: int
+    """"""
+    native_quantity_released: int
+    """"""
+    native_quantity_paid: int
+    """"""
+    native_fee_or_rebate: int
+    """"""
+    order_id: int
+    """"""
+    public_key: PublicKey
+    """"""
+    client_order_id: int
+    """"""
```

Comment:
Does it make sense to create a parent class for this? Almost half of the fields are the same in here.

Reply:
Parent class won't allow me to do https://github.com/serum-community/pyserum/pull/39/files/aa1ad637a1e8f48d1b5c9b336f9116d675f6ac65#diff-739bc899983a98df54f700f8da19b457R40.